components/layout/Login.js

```diff
@@ -120,10 +123,30 @@ const StyledLink = styled(Link)`
   border: 1px solid #ccc;
   border-radius: 4px;
   padding: 6px;
+  margin-right: 6px;
+  margin-left: 6px;
 `;
 const NavContainer = styled.div`
   display: flex;
   align-items: center;
   margin-bottom: 16px;
   justify-content: center;
 `;
+
+const StyledButton = styled.div`
```

Comment:
Just a little confusion, maybe name it StyledButtonContainer or something like that, needed a moment to check it! 🍬